library/src/main/java/com/github/javiersantos/bottomdialogs/BottomDialog.java

```diff
@@ -82,6 +86,25 @@ public void onClick(View view) {
                     bottomDialog.dismiss();
                 }
             });
+
+            if (builder.btn_colorPositive != 0) {
+                vPositive.setTextColor(builder.btn_colorPositive);
+            }
+
+            if (builder.btn_colorPositiveBackground == 0) {
+                TypedValue v = new TypedValue();
+                boolean hasColorPrimary = builder.context.getTheme().resolveAttribute(R.attr.colorPrimary, v, true);
```

Comment:
If there is no primary color assigned, use theme's `?attr/colorPrimary` as a background color.